tools/emscripten.py

```diff
@@ -1083,6 +1084,8 @@ def create_pointer_conversion_wrappers(metadata):
     '_wasmfs_unlink': '_p',
     '_wasmfs_mkdir': '_p_',
     '_wasmfs_open': '_p__',
+    '_wasmfs_mount': '_pp',
+    '_wasmfs_chdir': '_p',
```

Comment:
are the changes in this file related to the PR?

Reply:
Yes, these are needed to make some of the new test configurations work.